Changes for RabbitMQ transport v10 with ServiceControl (#7038)
Particular/docs.particular.net: servicecontrol/upgrades/6.4to6.5.md

---
title: Upgrade ServiceControl from Version 6.4.x to Version 6.5.x
summary: Instructions on how to upgrade ServiceControl from version 6.4.x to 6.5.x
reviewed: 2025-03-13
isUpgradeGuide: true
component: ServiceControl
---

## RabbitMQ delivery limit validation

> ServiceControl version 6.5.x uses rabbitMQ transport version 10.x.  When using RabbitMQ 4 and above the management API is required as part of the transport to validate the delivery limit of quorum queues to insure that it's set to `unlimited` for proper recoverability functionality.

Reviewer: ```suggestion
ServiceControl version 6.5.x uses RabbitMQ transport version 10.x.  This version of the transport uses the management API to validate that the delivery limit of each NServiceBus quorum queue is set to `unlimited`, so as not to interfere with NServiceBus' recoverability functionality.
```

> The [management API plugin](https://www.rabbitmq.com/docs/management#getting-started) is required to be installed on the node.  If the configuration of the management API is different than the default values based on the broker connection string, then the connection can be configured using the [RabbitMQ transport configuration options](/servicecontrol/transports.md#rabbitmq).

Reviewer: ```suggestion
The [management plugin](https://www.rabbitmq.com/docs/management#getting-started) is required to be installed on the node.  If the management API's host and/or user credentials are different than the values in the broker connection string, then they can be configured using the [RabbitMQ transport configuration options](/servicecontrol/transports.md#rabbitmq).
```
